content/swift/concepts/protocols/terms/codingkeyrepresentable/codingkeyrepresentable.md

```diff
@@ -0,0 +1,52 @@
+---
+Title: 'Protocols'
+Description: 'A protocol type that gives you the ability to convert it into a coding key and vice versa.'
+Subjects:
+  - 'iOS'
+  - 'Mobile Development'
+Tags:
+  - 'Protocols'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+
+The `CodingKeyRepresentable` protocol allows you to covert a CodingKey type between a personalized type. This protocol enables you to use dictionaries that are not a string or Int(non-standard keys) to be encoded using organized 'keyed' containers instead of 'unkeyed' containers.
+
+
+## Syntax
+```pseudo
+protocol CodingKeyRepresentable
+```
+
```

Comment:
Remove extra line. There should never be more than one empty line between content